### PR TITLE
New cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
PR contains mostly new instrument cal csvs from vendor refurb; also contains 2 NUTNR post-deploy cal csvs

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
Added 7 new FLORT-D csvs

File: calibration/PHSEND/CGINS-PHSEND-P0095__20200103.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0095,CC_ea434,17372.0, source_file PHSEN-D_SAMI-pH_SN_P0095_Validation_Certificate_2020-01-03.pdf
+P0095,CC_ea578,94.1,
+P0095,CC_eb434,2284.0,
+P0095,CC_eb578,38676.0,
+P0095,CC_ind_off,0,
+P0095,CC_ind_slp,1,
+P0095,CC_psal,35,Constant
```
Comment (reedan88) [inline calibration/PHSEND/CGINS-PHSEND-P0095__20200103.csv]: Looks good.

File: calibration/PHSEND/CGINS-PHSEND-P0096__20200103.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0096,CC_ea434,17372.0, source_file PHSEN-D_SAMI-pH_SN_P0096_Validation_Certificate_2020-01-03.pdf
+P0096,CC_ea578,94.1,
+P0096,CC_eb434,2284.0,
+P0096,CC_eb578,38676.0,
+P0096,CC_ind_off,0,
+P0096,CC_ind_slp,1,
+P0096,CC_psal,35,Constant
```
Comment (reedan88) [inline calibration/PHSEND/CGINS-PHSEND-P0096__20200103.csv]: Looks good.

File: calibration/PHSEND/CGINS-PHSEND-P0099__20200103.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0099,CC_ea434,17372.0, source_file PHSEN-D_SAMI-pH_SN_P0099_Validation_Certificate_2020-01-03.pdf
+P0099,CC_ea578,94.1,
+P0099,CC_eb434,2284.0,
+P0099,CC_eb578,38676.0,
+P0099,CC_ind_off,0,
+P0099,CC_ind_slp,1,
+P0099,CC_psal,35,Constant
```
Comment (reedan88) [inline calibration/PHSEND/CGINS-PHSEND-P0099__20200103.csv]: Looks good.

File: calibration/PHSEND/CGINS-PHSEND-P0155__20200106.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0155,CC_ea434,17372.0, source_file PHSEN-D_SAMI-pH_SN_P0155_Validation_Certificate_2020-01-06.pdf
+P0155,CC_ea578,94.1,
+P0155,CC_eb434,2284.0,
+P0155,CC_eb578,38676.0,
+P0155,CC_ind_off,0,
+P0155,CC_ind_slp,1,
+P0155,CC_psal,35,Constant
```
Comment (reedan88) [inline calibration/PHSEND/CGINS-PHSEND-P0155__20200106.csv]: Looks good.

**Comment by snwhite** [comment]
[Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-instrument-cal-csvs-dt-69)

**Comment by snwhite** [comment]
[Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-instrument-cal-csvs-dt-69)

File: calibration/FLORDG/CGINS-FLORDG-03304__20200122.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+3304,CC_angular_resolution,1.096,Constant; source_file FLORD-G_FLBBSB_SN_3304_Calibration_Files_2020-01-22.zip > FLBBSB-3304 Charsheets.pdf
+3304,CC_dark_counts_chlorophyll_a,0.082,in Volts (because analog signal going to CTDBP)
+3304,CC_dark_counts_volume_scatter,0.0820,in Volts (because analog signal going to CTDBP)
+3304,CC_depolarization_ratio,0.039,Constant
+3304,CC_measurement_wavelength,700,Constant
+3304,CC_scale_factor_chlorophyll_a,6,in Volts (because analog signal going to CTDBP)
+3304,CC_scale_factor_volume_scatter,1.320E-03,in Volts (because analog signal going to CTDBP)
+3304,CC_scattering_angle,140,Constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORDG/CGINS-FLORDG-03304__20200122.csv]: Looks good.

File: calibration/FLORDG/CGINS-FLORDG-04032__20200122.csv [hunk 6]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+4032,CC_angular_resolution,1.096,Constant; source_file FLORD-G_FLBBSB_SN_4032_Calibration_Files_2020-01-22.zip > FLBBSB-4032 Charsheets.pdf
+4032,CC_dark_counts_chlorophyll_a,0.074,in Volts (because analog signal going to CTDBP)
+4032,CC_dark_counts_volume_scatter,0.0740,in Volts (because analog signal going to CTDBP)
+4032,CC_depolarization_ratio,0.039,Constant
+4032,CC_measurement_wavelength,700,Constant
+4032,CC_scale_factor_chlorophyll_a,6,in Volts (because analog signal going to CTDBP)
+4032,CC_scale_factor_volume_scatter,1.329E-03,in Volts (because analog signal going to CTDBP)
+4032,CC_scattering_angle,140,Constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORDG/CGINS-FLORDG-04032__20200122.csv]: Looks good.

File: calibration/FLORDG/CGINS-FLORDG-04033__20200122.csv [hunk 7]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+4033,CC_angular_resolution,1.096,Constant; source_file FLORD-G_FLBBSB_SN_4033_Calibration_Files_2020-01-22.zip > FLBBSB-4033 Charsheets.pdf
+4033,CC_dark_counts_chlorophyll_a,0.050,in Volts (because analog signal going to CTDBP)
+4033,CC_dark_counts_volume_scatter,0.0500,in Volts (because analog signal going to CTDBP)
+4033,CC_depolarization_ratio,0.039,Constant
+4033,CC_measurement_wavelength,700,Constant
+4033,CC_scale_factor_chlorophyll_a,6,in Volts (because analog signal going to CTDBP)
+4033,CC_scale_factor_volume_scatter,1.254E-03,in Volts (because analog signal going to CTDBP)
+4033,CC_scattering_angle,140,Constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORDG/CGINS-FLORDG-04033__20200122.csv]: Looks good.

File: calibration/FLORDG/CGINS-FLORDG-04034__20200122.csv [hunk 8]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+4034,CC_angular_resolution,1.096,Constant; source_file FLORD-G_FLBBSB_SN_4034_Calibration_Files_2020-01-22.zip > FLBBSB-4034 Charsheets.pdf
+4034,CC_dark_counts_chlorophyll_a,0.071,in Volts (because analog signal going to CTDBP)
+4034,CC_dark_counts_volume_scatter,0.0710,in Volts (because analog signal going to CTDBP)
+4034,CC_depolarization_ratio,0.039,Constant
+4034,CC_measurement_wavelength,700,Constant
+4034,CC_scale_factor_chlorophyll_a,6,in Volts (because analog signal going to CTDBP)
+4034,CC_scale_factor_volume_scatter,1.276E-03,in Volts (because analog signal going to CTDBP)
+4034,CC_scattering_angle,140,Constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORDG/CGINS-FLORDG-04034__20200122.csv]: Looks good.

File: calibration/FLORTD/CGINS-FLORTD-01214__20200129.csv [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+1214,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1214_Calibration_Files_2020-01-29.zip > BBFL2W-1214_CharSheets.pdf
+1214,CC_dark_counts_cdom,43,
+1214,CC_dark_counts_chlorophyll_a,49,
+1214,CC_dark_counts_volume_scatter,48,
+1214,CC_depolarization_ratio,0.039,Constant
+1214,CC_measurement_wavelength,700,Constant
+1214,CC_scale_factor_cdom,0.0682,
+1214,CC_scale_factor_chlorophyll_a,0.0115,
+1214,CC_scale_factor_volume_scatter,2.166e-06,
+1214,CC_scattering_angle,124,Constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORTD/CGINS-FLORTD-01214__20200129.csv]: Looks good.

File: calibration/FLORTD/CGINS-FLORTD-01216__20200116.csv [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+1216,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1216_Calibration_Files_2020-01-16.zip > BBFL2W-1216_CharSheets.pdf
+1216,CC_dark_counts_cdom,50,
+1216,CC_dark_counts_chlorophyll_a,55,
+1216,CC_dark_counts_volume_scatter,51,
+1216,CC_depolarization_ratio,0.039,Constant
+1216,CC_measurement_wavelength,700,Constant
+1216,CC_scale_factor_cdom,0.0668,
+1216,CC_scale_factor_chlorophyll_a,0.0113,
+1216,CC_scale_factor_volume_scatter,2.229E-06,
+1216,CC_scattering_angle,124,Constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORTD/CGINS-FLORTD-01216__20200116.csv]: Looks good.

File: calibration/FLORTD/CGINS-FLORTD-01217__20200117.csv [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+1217,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1217_Calibration_Files_2020-01-17.zip > BBFL2W-1217 Charsheet.pdf
+1217,CC_dark_counts_cdom,54,
+1217,CC_dark_counts_chlorophyll_a,50,
+1217,CC_dark_counts_volume_scatter,42,
+1217,CC_depolarization_ratio,0.039,Constant
+1217,CC_measurement_wavelength,700,Constant
+1217,CC_scale_factor_cdom,0.0619,
+1217,CC_scale_factor_chlorophyll_a,0.0109,
+1217,CC_scale_factor_volume_scatter,2.335E-06,
+1217,CC_scattering_angle,124,Constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORTD/CGINS-FLORTD-01217__20200117.csv]: Looks good.

File: calibration/FLORTD/CGINS-FLORTD-01218__20200117.csv [hunk 12]
```diff
@@ -0,0 +1,11 @@
...
+1218,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1218_Calibration_Files_2020-01-17.zip > BBFL2W-1218 Charsheet.pdf
+1218,CC_dark_counts_cdom,49,
+1218,CC_dark_counts_chlorophyll_a,52,
+1218,CC_dark_counts_volume_scatter,51,
+1218,CC_depolarization_ratio,0.039,Constant
+1218,CC_measurement_wavelength,700,Constant
+1218,CC_scale_factor_cdom,0.0991,
+1218,CC_scale_factor_chlorophyll_a,0.0112,
+1218,CC_scale_factor_volume_scatter,2.573e-06,
+1218,CC_scattering_angle,124,Constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORTD/CGINS-FLORTD-01218__20200117.csv]: Looks good.  The case for the "e" shouldn't matter.  I see both in other cal files.

File: calibration/FLORTD/CGINS-FLORTD-01300__20200116.csv [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+1300,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1300_Calibration_Files_2020-01-16.zip > BBFL2W-1300_CharSheets.pdf
+1300,CC_dark_counts_cdom,50,
+1300,CC_dark_counts_chlorophyll_a,61,
+1300,CC_dark_counts_volume_scatter,52,
+1300,CC_depolarization_ratio,0.039,Constant
+1300,CC_measurement_wavelength,700,Constant
+1300,CC_scale_factor_cdom,0.0759,
+1300,CC_scale_factor_chlorophyll_a,0.0119,
+1300,CC_scale_factor_volume_scatter,1.907E-06,
+1300,CC_scattering_angle,124,Constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORTD/CGINS-FLORTD-01300__20200116.csv]: Looks good.

File: calibration/FLORTD/CGINS-FLORTD-01301__20200117.csv [hunk 14]
```diff
@@ -0,0 +1,11 @@
...
+1301,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1301_Calibration_Files_2020-01-17.zip > BBFL2W-1301 Charsheet.pdf
+1301,CC_dark_counts_cdom,57,
+1301,CC_dark_counts_chlorophyll_a,53,
+1301,CC_dark_counts_volume_scatter,55,
+1301,CC_depolarization_ratio,0.039,Constant
+1301,CC_measurement_wavelength,700,Constant
+1301,CC_scale_factor_cdom,0.0773,
+1301,CC_scale_factor_chlorophyll_a,0.0114,
+1301,CC_scale_factor_volume_scatter,2.220E-06,
+1301,CC_scattering_angle,124,Constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORTD/CGINS-FLORTD-01301__20200117.csv]: Looks good.

File: calibration/FLORTD/CGINS-FLORTD-01323__20200129.csv [hunk 15]
```diff
@@ -0,0 +1,11 @@
...
+1323,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1323_Calibration_Files_2020-01-29.zip > BBFL2W-1323_CharSheets.pdf
+1323,CC_dark_counts_cdom,46,
+1323,CC_dark_counts_chlorophyll_a,51,
+1323,CC_dark_counts_volume_scatter,51,
+1323,CC_depolarization_ratio,0.039,Constant
+1323,CC_measurement_wavelength,700,Constant
+1323,CC_scale_factor_cdom,0.0665,
+1323,CC_scale_factor_chlorophyll_a,0.0117,
+1323,CC_scale_factor_volume_scatter,2.407E-06,
+1323,CC_scattering_angle,124,Constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORTD/CGINS-FLORTD-01323__20200129.csv]: Looks good.

File: calibration/FLORTD/CGINS-FLORTD-01324__20200117.csv [hunk 16]
```diff
@@ -0,0 +1,11 @@
...
+1324,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1324_Calibration_Files_2020-01-17.zip > BBFL2W-1324 Charsheet.pdf
+1324,CC_dark_counts_cdom,49,
+1324,CC_dark_counts_chlorophyll_a,55,
+1324,CC_dark_counts_volume_scatter,50,
+1324,CC_depolarization_ratio,0.039,Constant
+1324,CC_measurement_wavelength,700,Constant
+1324,CC_scale_factor_cdom,0.0662,
+1324,CC_scale_factor_chlorophyll_a,0.0110,
+1324,CC_scale_factor_volume_scatter,2.469E-06,
+1324,CC_scattering_angle,124,Constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORTD/CGINS-FLORTD-01324__20200117.csv]: Looks good.

File: calibration/PHSEND/CGINS-PHSEND-P0155__20200106.csv (hunk 4 above)
Comment (snwhite) [reply calibration/PHSEND/CGINS-PHSEND-P0155__20200106.csv]: Looks good.

File: calibration/PHSEND/CGINS-PHSEND-P0099__20200103.csv (hunk 3 above)
Comment (snwhite) [reply calibration/PHSEND/CGINS-PHSEND-P0099__20200103.csv]: Looks good.

File: calibration/PHSEND/CGINS-PHSEND-P0096__20200103.csv (hunk 2 above)
Comment (snwhite) [reply calibration/PHSEND/CGINS-PHSEND-P0096__20200103.csv]: Looks good.

File: calibration/PHSEND/CGINS-PHSEND-P0095__20200103.csv (hunk 1 above)
Comment (snwhite) [reply calibration/PHSEND/CGINS-PHSEND-P0095__20200103.csv]: Looks good.

File: calibration/PCO2WB/CGINS-PCO2WB-C0097__20200108.csv [hunk 17]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0097,CC_cala,0.0701, source_file PCO2W-B_SAMI-CO2_SN_C0097_Calibration_Certificate_2020-01-08.pdf
+C0097,CC_calb,0.4697,
+C0097,CC_calc,-1.2637,
+C0097,CC_calt,9.6778,
+C0097,CC_ea434,19706,constant
+C0097,CC_ea620,34,constant
+C0097,CC_eb434,3073,constant
+C0097,CC_eb620,44327,constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/PCO2WB/CGINS-PCO2WB-C0097__20200108.csv]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01089__20200128.csv [hunk 18]
```diff
@@ -0,0 +1,8 @@
...
...273.48, 274.28, 275.08, 275.88, 276.68, 277.48, 278.28, 279.08, 279.88, 280.69, 281.49, 282.29, 283.09, 283.89, 284.69, 285.49, 286.29, 287.09, 287.89, 288.69, 289.49, 290.29, 291.09, 291.89, 292.69, 293.49, 294.29, 295.09, 295.89, 296.69, 297.49, 298.29, 299.09, 299.89, 300.69, 301.49, 302.29, 303.09, 303.89, 304.68, 305.48, 306.28, 307.08, 307.88, 308.68, 309.48, 310.28, 311.08, 311.88, 312.68, 313.47, 314.27, 315.07, 315.87, 316.67, 317.47, 318.27, 319.06, 319.86, 320.66, 321.46, 322.26, 323.05, 323.85, 324.65, 325.45, 326.24, 327.04, 327.84, 328.63, 329.43, 330.23, 331.03, 331.82, 332.62, 333.41, 334.21, 335.01, 335.8, 336.6, 337.39, 338.19, 338.99, 339.78, 340.58, 341.37, 342.17, 342.96, 343.76, 344.55, 345.34, 346.14, 346.93, 347.73, 348.52, 349.31, 350.11, 350.9, 351.69, 352.49, 353.28, 354.07, 354.87, 355.66, 356.45, 357.24, 358.03, 358.83, 359.62, 360.41, 361.2, 361.99, 362.78, 363.57, 364.36, 365.15, 365.94, 366.73, 367.52, 368.31, 369.1, 369.89, 370.68, 371.47, 372.26, 373.04, 373.83, 374.62, 375.41, 376.2, 376.98, 377.77, 378.56, 379.34, 380.13, 380.91, 381.7, 382.49, 383.27, 384.06, 384.84, 385.63, 386.41, 387.2, 387.98, 388.76, 389.55, 390.33, 391.11, 391.9, 392.68]",
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/NUTNRB/CGINS-NUTNRB-01089__20200128.csv]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01090__20200129.csv [hunk 19]
```diff
@@ -0,0 +1,8 @@
...
...272.23, 273.03, 273.84, 274.64, 275.44, 276.25, 277.05, 277.85, 278.66, 279.46, 280.26, 281.07, 281.87, 282.68, 283.48, 284.28, 285.09, 285.89, 286.7, 287.5, 288.31, 289.11, 289.92, 290.72, 291.52, 292.33, 293.13, 293.94, 294.74, 295.55, 296.35, 297.16, 297.97, 298.77, 299.58, 300.38, 301.19, 301.99, 302.8, 303.6, 304.41, 305.21, 306.02, 306.83, 307.63, 308.44, 309.24, 310.05, 310.85, 311.66, 312.47, 313.27, 314.08, 314.88, 315.69, 316.49, 317.3, 318.11, 318.91, 319.72, 320.52, 321.33, 322.14, 322.94, 323.75, 324.55, 325.36, 326.16, 326.97, 327.78, 328.58, 329.39, 330.19, 331, 331.8, 332.61, 333.42, 334.22, 335.03, 335.83, 336.64, 337.44, 338.25, 339.05, 339.86, 340.66, 341.47, 342.27, 343.08, 343.88, 344.69, 345.49, 346.3, 347.1, 347.91, 348.71, 349.52, 350.32, 351.13, 351.93, 352.74, 353.54, 354.34, 355.15, 355.95, 356.76, 357.56, 358.36, 359.17, 359.97, 360.77, 361.58, 362.38, 363.18, 363.99, 364.79, 365.59, 366.4, 367.2, 368, 368.8, 369.61, 370.41, 371.21, 372.01, 372.81, 373.61, 374.42, 375.22, 376.02, 376.82, 377.62, 378.42, 379.22, 380.02, 380.82, 381.62, 382.43, 383.23, 384.03, 384.83, 385.62, 386.42, 387.22, 388.02, 388.82, 389.62, 390.42, 391.22, 392.02, 392.81, 393.61]",
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/NUTNRB/CGINS-NUTNRB-01090__20200129.csv]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01091__20200128.csv [hunk 20]
```diff
@@ -0,0 +1,8 @@
...
...9, 272.82, 273.64, 274.46, 275.28, 276.1, 276.93, 277.75, 278.57, 279.39, 280.21, 281.04, 281.86, 282.68, 283.5, 284.32, 285.15, 285.97, 286.79, 287.61, 288.43, 289.26, 290.08, 290.9, 291.72, 292.54, 293.37, 294.19, 295.01, 295.83, 296.65, 297.47, 298.3, 299.12, 299.94, 300.76, 301.58, 302.4, 303.22, 304.05, 304.87, 305.69, 306.51, 307.33, 308.15, 308.97, 309.79, 310.61, 311.43, 312.26, 313.08, 313.9, 314.72, 315.54, 316.36, 317.18, 318, 318.82, 319.64, 320.46, 321.28, 322.1, 322.92, 323.74, 324.55, 325.37, 326.19, 327.01, 327.83, 328.65, 329.47, 330.29, 331.1, 331.92, 332.74, 333.56, 334.38, 335.19, 336.01, 336.83, 337.65, 338.46, 339.28, 340.1, 340.92, 341.73, 342.55, 343.37, 344.18, 345, 345.81, 346.63, 347.44, 348.26, 349.08, 349.89, 350.71, 351.52, 352.34, 353.15, 353.96, 354.78, 355.59, 356.41, 357.22, 358.03, 358.85, 359.66, 360.47, 361.28, 362.1, 362.91, 363.72, 364.53, 365.34, 366.16, 366.97, 367.78, 368.59, 369.4, 370.21, 371.02, 371.83, 372.64, 373.45, 374.26, 375.07, 375.88, 376.69, 377.49, 378.3, 379.11, 379.92, 380.73, 381.53, 382.34, 383.15, 383.95, 384.76, 385.57, 386.37, 387.18, 387.98, 388.79, 389.59, 390.4, 391.2, 392.01, 392.81, 393.61, 394.42, 395.22, 396.02]",
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/NUTNRB/CGINS-NUTNRB-01091__20200128.csv]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01094__20200128.csv [hunk 21]
```diff
@@ -0,0 +1,8 @@
...
..., 271.85, 272.66, 273.47, 274.27, 275.08, 275.89, 276.7, 277.51, 278.32, 279.13, 279.94, 280.75, 281.56, 282.37, 283.18, 283.99, 284.8, 285.61, 286.42, 287.23, 288.03, 288.84, 289.65, 290.46, 291.27, 292.08, 292.89, 293.7, 294.51, 295.32, 296.12, 296.93, 297.74, 298.55, 299.36, 300.17, 300.98, 301.78, 302.59, 303.4, 304.21, 305.02, 305.82, 306.63, 307.44, 308.25, 309.06, 309.86, 310.67, 311.48, 312.29, 313.09, 313.9, 314.71, 315.51, 316.32, 317.13, 317.93, 318.74, 319.55, 320.35, 321.16, 321.97, 322.77, 323.58, 324.38, 325.19, 326, 326.8, 327.61, 328.41, 329.22, 330.02, 330.83, 331.63, 332.44, 333.24, 334.04, 334.85, 335.65, 336.46, 337.26, 338.06, 338.87, 339.67, 340.47, 341.28, 342.08, 342.88, 343.68, 344.49, 345.29, 346.09, 346.89, 347.69, 348.5, 349.3, 350.1, 350.9, 351.7, 352.5, 353.3, 354.1, 354.9, 355.7, 356.5, 357.3, 358.1, 358.9, 359.7, 360.5, 361.29, 362.09, 362.89, 363.69, 364.49, 365.28, 366.08, 366.88, 367.67, 368.47, 369.27, 370.06, 370.86, 371.66, 372.45, 373.25, 374.04, 374.84, 375.63, 376.42, 377.22, 378.01, 378.81, 379.6, 380.39, 381.19, 381.98, 382.77, 383.56, 384.35, 385.15, 385.94, 386.73, 387.52, 388.31, 389.1, 389.89, 390.68, 391.47, 392.26, 393.05, 393.84]",
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/NUTNRB/CGINS-NUTNRB-01094__20200128.csv]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01088__20200123.csv [hunk 22]
```diff
@@ -0,0 +1,8 @@
...
...73.7, 274.52, 275.34, 276.16, 276.99, 277.81, 278.63, 279.45, 280.27, 281.1, 281.92, 282.74, 283.56, 284.38, 285.21, 286.03, 286.85, 287.67, 288.49, 289.31, 290.14, 290.96, 291.78, 292.6, 293.42, 294.24, 295.07, 295.89, 296.71, 297.53, 298.35, 299.17, 299.99, 300.81, 301.63, 302.46, 303.28, 304.1, 304.92, 305.74, 306.56, 307.38, 308.2, 309.02, 309.84, 310.66, 311.48, 312.3, 313.12, 313.94, 314.76, 315.58, 316.4, 317.22, 318.04, 318.86, 319.68, 320.5, 321.32, 322.13, 322.95, 323.77, 324.59, 325.41, 326.23, 327.04, 327.86, 328.68, 329.5, 330.32, 331.13, 331.95, 332.77, 333.58, 334.4, 335.22, 336.04, 336.85, 337.67, 338.48, 339.3, 340.12, 340.93, 341.75, 342.56, 343.38, 344.19, 345.01, 345.82, 346.64, 347.45, 348.27, 349.08, 349.89, 350.71, 351.52, 352.34, 353.15, 353.96, 354.77, 355.59, 356.4, 357.21, 358.02, 358.84, 359.65, 360.46, 361.27, 362.08, 362.89, 363.7, 364.51, 365.32, 366.13, 366.94, 367.75, 368.56, 369.37, 370.18, 370.99, 371.8, 372.61, 373.41, 374.22, 375.03, 375.84, 376.64, 377.45, 378.26, 379.06, 379.87, 380.67, 381.48, 382.29, 383.09, 383.89, 384.7, 385.5, 386.31, 387.11, 387.91, 388.72, 389.52, 390.32, 391.13, 391.93, 392.73, 393.53, 394.33, 395.13, 395.93, 396.73]",
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/NUTNRB/CGINS-NUTNRB-01088__20200123.csv]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01103__20200123.csv [hunk 23]
```diff
@@ -0,0 +1,8 @@
...
...272.64, 273.44, 274.25, 275.05, 275.85, 276.66, 277.46, 278.26, 279.07, 279.87, 280.67, 281.48, 282.28, 283.08, 283.89, 284.69, 285.49, 286.3, 287.1, 287.91, 288.71, 289.51, 290.32, 291.12, 291.92, 292.73, 293.53, 294.33, 295.14, 295.94, 296.74, 297.55, 298.35, 299.15, 299.96, 300.76, 301.56, 302.37, 303.17, 303.97, 304.78, 305.58, 306.38, 307.19, 307.99, 308.79, 309.59, 310.4, 311.2, 312, 312.81, 313.61, 314.41, 315.21, 316.02, 316.82, 317.62, 318.42, 319.22, 320.03, 320.83, 321.63, 322.43, 323.23, 324.04, 324.84, 325.64, 326.44, 327.24, 328.04, 328.84, 329.64, 330.45, 331.25, 332.05, 332.85, 333.65, 334.45, 335.25, 336.05, 336.85, 337.65, 338.45, 339.25, 340.05, 340.85, 341.65, 342.45, 343.25, 344.04, 344.84, 345.64, 346.44, 347.24, 348.04, 348.84, 349.63, 350.43, 351.23, 352.03, 352.82, 353.62, 354.42, 355.22, 356.01, 356.81, 357.61, 358.4, 359.2, 360, 360.79, 361.59, 362.38, 363.18, 363.97, 364.77, 365.56, 366.36, 367.15, 367.95, 368.74, 369.53, 370.33, 371.12, 371.92, 372.71, 373.5, 374.29, 375.09, 375.88, 376.67, 377.46, 378.26, 379.05, 379.84, 380.63, 381.42, 382.21, 383, 383.79, 384.58, 385.37, 386.16, 386.95, 387.74, 388.53, 389.32, 390.11, 390.9, 391.68, 392.47, 393.26]",
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/NUTNRB/CGINS-NUTNRB-01103__20200123.csv]: Looks good.

**Pull request merged**